CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 Not yet released; provisionally v1.4.0 (may change).
 
+### Remove Unsupported Hashers
+
+The following hashers have been removed because of security vulnerabilities and
+performance issues:
+
+- `OBJECT_RFC6962_SHA256` - Security vulnerability
```

Comment:
A link to some external info on this security vulnerability would be useful, if it exists.